remote-api/src/main/java/org/alfresco/rest/api/impl/NodesImpl.java

```diff
@@ -649,6 +649,11 @@ private NodeRef getParentNodeRef(NodeRef nodeRef)
         return nodeService.getPrimaryParent(nodeRef).getParentRef();
     }
 
+    public NodeRef validateOrLookupNode(String nodeId)
```

Comment:
That could as well be a default method in the interface.

Reply:
Another good idea - thanks.